face_model/argface_model/argface_extract_features.py

```
@@ -27,61 +17,43 @@ def __init__(self, data_path):
         self.label_map = {}
-        logger.info("FeatureExtractor initialized with data path: %s", self.data_path)
 
     def extract_labels(self):
         try:
             for label, person in enumerate(os.listdir(self.data_path)):
                 self.label_map[label] = person
-            logger.info(f"Labels extracted: {self.label_map}")
         except Exception as e:
-            logger.error("Error extracting labels: %s", e)
-            raise
+            raise ValueError("Error extracting labels: %s", e)
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)